README.md

```diff
@@ -47,109 +66,119 @@ Here is the complete list of pre-requisites:
 - Access to OpenShift 4 cluster
 
 ### Build
+To build the operator use:
 ```
 make build
 ```
 ### Test
-* run unit test:
-```
-make test
-```
-* run e2e test:
-For running e2e tests, have minishift started.
-```
-make test-e2e-local
-```
-> Note: e2e test will deploy operator in project `devconsole-e2e-test`, if your tests timeout and you wan to debug:
-> - oc project devconsole-e2e-test
-> - oc get deployment,pod
-> - oc logs pod/devconsole-operator-5b4bbc7d-4p7hr
-
-## Deployment
+* To run unit test use:
+    ```
+    make test
+    ```
+* To run e2e test:
+
+    Start Minishift and run:
+    ```
+    make test-e2e-local
+    ```
+
+    **Note:**
+The e2e test deploys the operator in the project `devconsole-e2e-test`. If your
+tests timeout and you want to debug, run:
+    ```
+    oc project devconsole-e2e-test
+    oc get deployment,pod
+    oc logs pod/devconsole-operator-5b4bbc7d-4p7hr
```

Comment:
Is this the specific pod each time? cc @baijum

Reply:
Added syntax for Variable